apps/settings/js/panels/wifi/panel.js

```diff
@@ -25,6 +25,7 @@ define(function(require) {
           panel: panel,
           wifi: panel,
           wpsColumn: panel.querySelector('.wps-column'),
+          wifiSleepModeStatus: panel.querySelector('#wifi-sleepMode input'),
```

Comment:
When specifying the "name" attribute with the name of a settings field on an input, settings app binds the settings field to the input automatically, so all changes made to this script are not required.